modules/tinymce/src/core/main/ts/api/util/VK.ts

```diff
@@ -19,6 +19,7 @@ interface VK {
   DELETE: number;
   DOWN: number;
   ENTER: number;
+  ESCAPE: number;
```

Comment:
We said in the call this should be called `ESC` instead of `ESCAPE`?

- [ ] Rename to `ESC`

Reply:
Also this will need a changelog entry to say we've added it to the `VK` API.

Reply:
I didn't hear about calling it `ESC` in the call, but I definitely prefer `ESCAPE` (we also call it `escape` in Agar).

Reply:
I mentioned it in the call and we said yes at the time. I largely mentioned it because this won't be minified since it's on the public API and the actual key on keyboards is `Esc`.

Reply:
I made this change, but I do really feel it should have been `ESCAPE` for consistency (e.g. `DELETE`, `PAGE_DOWN`, `SPACEBAR`).

Reply:
To be clear, this doesn't have to be `ESC`, I was just pointing out it's different to what had been mentioned in the call as what we should do. Maybe lets ask at standup today again, as it's likely others weren't paying much attention either yesterday afternoon.